src/rt/util/hash.d

```diff
@@ -33,10 +33,14 @@ size_t hashOf( const(void)[] buf, size_t seed )
         version( HasUnalignedOps )
         {
             if (!__ctfe)
+            {
                 return *cast(ushort*) x;
+            }
```

Comment:
In the case of `version(HasUnalignedOps)`, and `__ctfe == true`, this function has no body.